experimental/trusted_shuffler/scripts/deploy

```diff
@@ -10,4 +10,7 @@ gcloud run deploy "${DEPLOYMENT}" \
     --region=europe-west1 \
     --image="${DOCKER_IMAGE}" \
     --use-http2 \
-    --allow-unauthenticated
+    --allow-unauthenticated \
+    --set-env-vars "BACKEND_URL=http://localhost:8888" \
```

Comment:
```suggestion
    --set-env-vars=BACKEND_URL=http://localhost:8888 \
```

(I don't think the quotes are necessary)

Reply:
Done.